tests/acceptance/ember-keyboard-test-auto-responder-test.js

```diff
@@ -0,0 +1,176 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { click, fillIn, blur, visit } from '@ember/test-helpers';
+import { gte } from 'ember-compatibility-helpers';
+import config from 'dummy/config/environment';
+
+import {
+  mouseDown,
+  keyUp,
+  keyDown,
+  touchStart,
+  keyDownWithElement,
+} from 'ember-keyboard/test-support/test-helpers';
+
+import { hook } from '../helpers/hook';
+
+import {
+  getValues,
+  getMouseValues,
+  getTouchValues,
+} from '../helpers/get-values';
+
+module('Acceptance | ember keyboard test auto responder', function (hooks) {
```

Comment:
```suggestion
module('Acceptance | disableOnInputFields config', function (hooks) {
```

...and update filename to match.